.github/workflows/health_checks.yml

```diff
@@ -512,6 +512,10 @@ jobs:
             node-version: 20
```

Comment:
should we remove old reference to 20 here?

Reply:
I saw this and left it alone. One can still trigger workflow with "20" included in parameter list, if this is removed the exclusion won't work.